tests/ccl_test_pyccl_interface.py

```diff
@@ -617,20 +619,9 @@ def check_cls_nu(cosmo):
     assert_( all_finite(ccl.angular_cl(cosmo, nc1, lens1, ell_arr)) )
     assert_( all_finite(ccl.angular_cl(cosmo, nc1, lens2, ell_arr)) )
 
-    # Check get_internal_function()
-    a_scl = 0.5
-    a_lst = [0.2, 0.4, 0.6, 0.8, 1.]
-    a_arr = np.linspace(0.2, 1., 5)
-    assert_( all_finite(nc1.get_internal_function(cosmo, 'dndz', a_scl)) )
-    assert_( all_finite(nc1.get_internal_function(cosmo, 'dndz', a_lst)) )
-    assert_( all_finite(nc1.get_internal_function(cosmo, 'dndz', a_arr)) )
```

Comment:
So we lost this method? Not a big deal, but we should think about adding it back. I'd be nice to be able to access to the values of the kernels used by the C layer. Maybe make an issue for this?

Reply:
It should be easy to add `eval_kernel`, `eval_transfer` and `eval_f_ell` functions here. I'll try to get this done in the next few commits and otherwise will open an issue.